mailer.js

```diff
@@ -263,6 +255,52 @@ module.exports = function (log) {
     })
   }
 
+
+  Mailer.prototype.authorizeLoginEmail = function (message) {
+    log.trace({ op: 'mailer.authoirzeLoginEmail', email: message.email, uid: message.uid })
+
+    var templateName = 'authorizeLoginEmail'
+    var query = {
+      code: message.code,
+      email: message.email,
+      uid: message.uid
+    }
+
+    if (message.service) { query.service = message.service }
```

Comment:
Yeah, please. Since there is no longer a link to click on, service, redirectTo and resume are no longer needed.